libs/blocks/global-navigation/global-navigation.css

```diff
@@ -461,6 +461,17 @@ header.global-navigation {
   right: auto;
 }
 
+/* Mobile styles */
+@media (max-width: 900px) {
+  .feds-menu-section > a:first-child {
+    margin-top: 12px;
+  }
+
+  .feds-menu-column > ul {
+    padding: 12px 0;
+  }
+}
```

Comment:
With the risk of adding a few more lines of CSS, I think I'd rather stick to the `min-width` pattern that we use throughout our styles. As in - declare it for mobile, cancel it for desktop if needed (which in this case it is).

Reply:
Sure! That was my first option, but I chose this to keep the CSS lines to a minimum. I changed the code to preserve the consistency in pattern 👍